ci/credentials.yml.example

```diff
@@ -9,10 +9,10 @@ aws_access_key_id: ACCESS_KEY_ID
 aws_secret_access_key: SECRET_ACCESS_KEY
 aws_account_id: XXXXXXXXXXXX
 aws_partition: aws
-aws_default_region: us-east-1
+aws_default_region: us-gov-west-1
 aws_s3_tfstate_bucket: MY_TERRAFORM_BUCKET
-aws_az1: us-east-1b
-aws_az2: us-east-1d
+aws_az1: us-gov-west-1b
+aws_az2: us-gov-west-1d
```

Comment:
1d is not a thing on govcloud. There's just a and b.

Reply:
Missed this earlier — thanks for the edit.